src/pages/PageSearchResult.vue

```diff
@@ -60,13 +60,21 @@
         {{ results.total }} results for
         <strong>{{ this.$route.query.query }}</strong>
       </p>
-      <SearchResultList :results="results"></SearchResultList>
+      <SearchResultList :dinos="results.dinos"></SearchResultList>
+      <LoadingSpinner v-if="loading"></LoadingSpinner>
+      <Button
+        v-else-if="results.dinos.length < results.total"
+        class="search-results__button-more"
+        @click="fetchMore"
+        >more</Button
```

Comment:
maybe make this Load More to match capitalisation/content in other action buttons?